x/authz/keeper/keys.go

```diff
@@ -45,3 +45,10 @@ func addressesFromGrantStoreKey(key []byte) (granterAddr, granteeAddr sdk.AccAdd
 
 	return granterAddr, granteeAddr
 }
+
+func granteeFromGranterPrefixedGrantStoreKey(key []byte) (granteeAddr sdk.AccAddress) {
+	granteeAddrLen := key[0] // remove prefix key
```

Comment:
I think this comment is poorly written. It doesn't remove anything. The first byte of this byte string is the length. It's used correctly as far as I can tell. If anything, the comment should just indicate "first byte is the length"

Reply:
Done